CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
-## [Ongoing](https://github.com/pbrisbin/downgrade/compare/v8.1.1...master)
+## [Ongoing](https://github.com/pbrisbin/downgrade/compare/v8.2.0...master)
 
 None
 
+## [v8.2.0](https://github.com/pbrisbin/downgrade/compare/v8.1.1...v8.2.0)
+
+- [NEW] Pacman cache directory accumulation to support multiple cache locations
+- [NEW] Removal of --nosudo CLI option, downgrade now expects root privileges 
```

Comment:
This feels like a big change, the entire interface is completely changed. I was thinking of `v9` if only to signal that. On the other hand, misuse is not harmful, the error is clear, and the behavior once executed has not changed in a meaningful way... So I'm fine with only a minor bump.

Reply:
Hmm true, it is indeed a big change and a major bump would definitely indicate that. I had qualms about consecutive major bumps taking place in the last few releases but I guess the changes are what they are and the versioning should reflect that. Major bump it is :)

Reply:
FWIW, I treat Arch packages very differently than almost any other versioning situation. In a library, doing a major version update probably results in a large set of users that will refuse to move up and a maintenance headache if you've committed to supporting older versions. In Arch, everyone will just `-Syu` and it'll be fine. I still try to apply the same rules about indicating kinds of changes, but there's zero hesitation about large jumps in version.